dev/cognitiveservices/data-plane/Language/analyzetext-authoring.json

```diff
@@ -3209,7 +2590,8 @@
         "lastTrainedDateTime",
         "lastTrainingDurationInSeconds",
         "modelExpirationDate",
-        "modelTrainingConfigVersion"
+        "modelTrainingConfigVersion",
+        "hasSnapshot"
```

Comment:
Adding a required property is a breaking change.

Reply:
Actually, I see this is a response-only model so adding a required property only requires a new api-version, which you're doing. Sorry.